pkgs/by-name/wi/wipe/package.nix

```
@@ -22,12 +27,12 @@ stdenv.mkDerivation rec {
 
   patches = [ ./fix-install.patch ];
 
-  meta = with lib; {
+  meta = {
     description = "Secure file wiping utility";
-    mainProgram = "wipe";
     homepage = "https://wipe.sourceforge.net/";
-    license = licenses.gpl2Plus;
-    platforms = platforms.all;
-    maintainers = [ maintainers.abbradar ];
+    license = lib.licenses.gpl2Plus;
+    platforms = lib.platforms.all;
```

```suggestion
    platforms = lib.platforms.unix;
```
Does not cross-compile to Windows https://paste.fliegendewurst.eu/E3bUid.log